src/common/components/linkButton/LinkButton.tsx

```diff
@@ -22,14 +23,18 @@ function LinkButton({
   component,
   variant,
   rel,
+  iconRight,
 }: Props) {
+  const iconRightWithPadding = iconRight ? ` ${iconRight}` : '';
+
```

Comment:
If the `iconRight` is made a string, it will display as 

![image](https://user-images.githubusercontent.com/25960123/105003698-aef82380-5a3b-11eb-854a-205a952025a4.png)

If the icon is used as it is, it leaves a nice padding between the text and icon.

Reply:
Ah, correct. I forgot to check this change in the browser.